README.md

```diff
@@ -113,6 +113,10 @@ Or download [latest archive](https://github.com/php-censor/php-censor/releases/l
 ```bash
 # For Debian-based
 aptitude install beanstalkd
+# Check if the service is running:
```

Comment:
I guess for Debian-based distributives will be more correct commands: `systemctl status beanstalkd.service`, `systemctl enable beanstalkd.service` and `systemctl start beanstalkd.service`.

Reply:
I tried on a new Docker image for Debian Buster (=10), but there I got the error message "bash: systemctl: command not found". I guess this is because I have the wrong Docker image, but question then becomes what the correct package is.

I tried "/etc/init.d/beanstalkd start" in this same image, and this worked, without errors.